PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -349,7 +349,9 @@ void FindReplaceDlg::create(int dialogID, bool isRTL, bool msgDestParent, bool t
 	if (nppGUI._findWindowPos.bottom - nppGUI._findWindowPos.top != 0)  // check height against 0 as a test of valid data from config
 	{
 		RECT rc = getViewablePositionRect(nppGUI._findWindowPos);
-		::SetWindowPos(_hSelf, HWND_TOP, rc.left, rc.top, rc.right - rc.left, rc.bottom - rc.top, swpFlags);
+		::SetWindowPos(_hSelf, HWND_TOP, rc.left, rc.top, 0, 0, SWP_HIDEWINDOW | SWP_NOSIZE | SWP_NOACTIVATE);
+		::SetWindowPos(_hSelf, HWND_TOP, 0, 0, _rc.right - _rc.left, _rc.bottom - _rc.top, swpFlags | SWP_NOMOVE);

```

Comment:
@donho 
currently it resets saved dialog size to min size.
To used saved size, more works need to be done. Like saving unscaled size and than scale it or saving used dpi.

Reply:
@ozone10 
I think this behaviour is OK - it's original behaviour.

Reply:
Upvote: It is a good way to reset position and size of the dialog. Please keep it this way :-)